### PR TITLE
Adding PortugolSyntax package

### PR DESCRIPTION
**Pull request by rlsantos**

### CODE REVIEW

**Comment by jrappen** [comment]
You might want to add a github actions workflow, compare: <https://github.com/sublimehq/Packages/blob/master/.github/workflows/ci.yml>.

**Comment by rlsantos** [comment]
Thanks, at this time, it sounds good for me, any suggestions?

**Comment by rlsantos** [comment]
> You might want to add a github actions workflow, compare: https://github.com/sublimehq/Packages/blob/master/.github/workflows/ci.yml.

Do I need change something? This package will help my students to understand and learn to program, then They are so eager for it.

File: repository/p.json [hunk 1]
```diff
@@ -1866,6 +1866,18 @@
 				}
 			]
 		},
+		{
+			"name": "Portugol Syntax",
+			"details": "https://github.com/rlsantos/PortugolSyntax",
+			"readme": "https://github.com/rlsantos/PortugolSyntax/blob/main/README.md",
```
Comment (braver) [inline repository/p.json:1872]: Since your readme is in the default location, you don't need to link to it here (ie. please remove this line)

**Comment by braver** [comment]
> Do I need change something

You don't _need_ the ci, but syntaxes are much easier to develop with tests, and CI is a good thing to set up to automate that.

**Comment by jrappen** [comment]
There's also <https://github.com/marketplace/actions/sublime-text-syntax-tests>.

**Comment by braver** [comment]
Please respond to the feedback to continue this PR.

**Pull request closed without merging**